### ISSUE TITLE
Issue on page /aiml-common/lectures/nlp/rnn-language-models/_index.html

### ISSUE THREAD

**Issue by Tommyhu28** [pantelis/artificial-intelligence#8]
> To train an LSTM language model
> 
> We start with big corpus of text which is a sequence of tokens 
>  where T is the number of words / tokens in the corpus.
> 
> Every time step we feed one word at a time to the LSTM and compute the output probability distribution **Missing argument for \mathbf,** which is, by construction, a conditional probability distribution of every word in the dictionary given the words we have seen so far.
> 
> The loss function at time step  is the classic cross entropy loss between the predicted probability distribution and the distribution that corresponds to the one-hot encoded true next token. **$Missing argument for \mathbf$**
> 
> Average all the t-step losses
